File: dev-mode/sbt-plugin/src/sbt-test/routes-compiler-plugin/injected-routes-compilation-with-request-passed/app/controllers/module/ModuleController.java
/*
 * Copyright (C) 2009-2019 Lightbend Inc. <https://www.lightbend.com>
 */
package controllers.module;

import play.mvc.*;

public class ModuleController extends Controller {
  public Result index(Http.Request req) {
    return ok(req.uri());
  }
}

File: dev-mode/sbt-plugin/src/sbt-test/play-sbt-plugin/routes-compiler-injected-routes-compilation/app/controllers/module/ModuleController.scala
/*
 * Copyright (C) Lightbend Inc. <https://www.lightbend.com>
 */

package controllers.module

import play.api.mvc._
import javax.inject.Inject

class ModuleController @Inject()(c: ControllerComponents) extends AbstractController(c) {
  def index = Action {
    Ok
  }
}
